src/OpenApi/ClientGenerator.php

```diff
@@ -18,13 +18,16 @@ class ClientGenerator
         'boolean' => 'bool',
         'integer' => 'int',
         'float' => 'float',
-        'number' => 'float'
+        'number' => 'float',
+        'file' => 'string',
     ];
 
     public function __construct()
     {
         $retailer = (new SwaggerSpecs())->load(__DIR__ . '/retailer.json')
-            ->merge((new SwaggerSpecs())->load(__DIR__ . '/shared.json'));
+            ->merge((new SwaggerSpecs())->load(__DIR__ . '/retailer-v10.json'))
+            ->merge((new SwaggerSpecs())->load(__DIR__ . '/shared.json'))
+            ->merge((new SwaggerSpecs())->load(__DIR__ . '/shared-v10.json'));
```

Comment:
My intention was that the latest API specs could be downloaded to `retailer.json` and `shared.json`. With this change V8 and V10 specs are merged. Is there any reason to do it like this?

Reply:
this was only temperary for testing, because v10 is still in beta. and the json of that api contains (for now) only there changed beta endpoints. 

[Retailer API specifications](https://api.bol.com/retailer/public/apispec/Retailer%20API%20-%20v10)

Reply:
this was only temperary for testing, because v10 is still in beta. and the json of that api contains (for now) only there changed beta endpoints.

[Retailer API specifications](https://api.bol.com/retailer/public/apispec/Retailer%20API%20-%20v10)